pom.xml

```diff
@@ -25,6 +25,16 @@
         <tag>HEAD</tag>
     </scm>
 
+    <distributionManagement>

```

Comment:
Do we really need to add `<distributionManagement>` in the parent pom?

Reply:
For now we need to add this attribute, or jenkins job will throw exception when we release snapshot versions.
``` powershell
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-deploy-plugin:2.7:deploy (default-deploy) on project azure-maven-plugins: Deployment failed: repository element was not specified in the POM inside distributionManagement element or in -DaltDeploymentRepository=id::layout::url parameter -> [Help 1]
```
We may combine the Aggregate POM and Parent POM , tracking in issue #453